src/System.AppContext/pkg/System.AppContext.pkgproj

```diff
@@ -14,8 +19,16 @@
     <ProjectReference Include="..\src\System.AppContext.csproj">
       <TargetGroup>net462</TargetGroup>
```

Comment:
You should directly reference System.AppContext.builds file now since they are all included in here. That will also help keep these things consistent.